### DISCUSSION TITLE
What code do you use to set no style on the app?

### DISCUSSION THREAD

**Discussion by Anixx** [otya128/winevdm#1294] [General]
What code do you use to disable the visual style, specifically the visual style of the window frame?

The usual method SetThemeAppProperties(0); will make the window frame Aero Lite. 
How do you do this?

**Comment by otya128** [comment]
https://github.com/otya128/winevdm/blob/4a671f098a46470645d236169d7c4ee281b7826d/user/message.c#LL4759C1-L4763C1

**Reply by Anixx** [reply]
Thank you very much!
